kore/src/Kore/Exec.hs

```diff
@@ -448,7 +458,19 @@ mergeRules ruleMerger verifiedModule ruleNames =
                 rules <- extractRules rewriteRules ruleNames
                 case rules of
                     [] -> Left "Empty rule list."
-                    (r : rs) -> Right (r :| rs)
+                    (r : rs) -> case maybeInitialConfiguration of
+                        Nothing -> Right (r :| rs)
+                        Just cfg -> Right (identityRule cfg :| r : rs)
+
+            identityRule :: TermLike Variable -> RewriteRule Variable
```

Comment:
The `identityRule` is because you can only insert rules, not configurations?

Reply:
As discussed on Slack, I'm rolling this back.

To answer your question, I could have changed the merging algorithms to take an optional configuration, or to take a mixed list of configurations and rules, but then I realized that, while we are starting with a list of rules of the type `a=>b` where `a` is being rewritten in one step to `b`, when the algorithm is running we are merging rules where the rewriting is being done in any number `n>0` of steps. So then the obvious extension was to also include rules that rewrite in `0` steps, a.k.a. (partial) configurations, which makes `identityRule` the natural representation here.